README.md

```diff
@@ -16,27 +16,40 @@ To install the operator courier project to use from the command line, just insta
 $ pip3 install operator-courier
 ```
 
-Once the project is installed, you can verify that a set of files is valid and can be bundled and pushed. First, create a flat directory containing all of the CSV, CRD and PACKAGE files that are included in your bundle.  Then just use `operator-courier verify` to test it.
+Once the project is installed, you can run `verify` command on a set of operator manifests with different versions in the source directory. The source directory includes a package file in the source directory root, and version folders containing CRD and CSV files required for that manifest version (See [Manifest format](https://github.com/operator-framework/operator-registry#manifest-format) for more details)
```

Comment:
This is a very convoluted sentence. I recommend rewriting it to just reference the manifest format:
>Once the project is installed, you can run **the** `verify` command on a directory that adheres to the expected [Manifest format](https://github.com/operator-framework/operator-registry#manifest-format).

Reply:
@awgreene Thanks for the suggestion! Updated.